src/components/ChartComponent/ChartComponent.scss

```diff
@@ -2,3 +2,19 @@
   position: relative;
   height: 600px;
 }
+
+@media screen and (min-width: 415px) {
+  .line-chart {
+    height: 600px;
+  }
+}
+
+@media screen and (max-width: 414px) {
+  .chart-container {
+    height: 430px
```

Comment:
I think this one is overdone. try to remove it ;) 

`.line-chart` as a child of this container, should expand `.chart-container` height automatically :)